### PR TITLE
[EXPLORER] Cleanup after last commit CORE-15369

### PR DESCRIPTION
**Pull request by JoachimHenze**
Improves code that was added by 0.4.15-dev-5078-g 46de3a22acc964f27a5260e144089125f83586ff without any review yesterday.

JIRA issue: [CORE-15369](https://jira.reactos.org/browse/CORE-15369)

### CODE REVIEW

**Review by katahiromz — APPROVED** [review]

File: base/shell/explorer/traywnd.cpp [hunk 1]
```diff
@@ -3045,10 +3045,8 @@ class CTrayWindow :
 
     LRESULT OnNcLButtonUp(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
     {
-        if (CheckShowDesktopButtonClick(lParam, bHandled))
-            return 0;
-
-        return 0;
+        CheckShowDesktopButtonClick(lParam, bHandled);
+        return FALSE;
```
Comment (learn-more) [inline base/shell/explorer/traywnd.cpp:3049]: Why FALSE?
The return is an LRESULT, not a BOOLEAN

Comment (binarymaster) [reply base/shell/explorer/traywnd.cpp:3049]: I think this is following existing code in this file. There are other functions which were coded the same way.

**Review by HBelusca — APPROVED** [review]

File: base/shell/explorer/traywnd.cpp (hunk 1 above)
Comment (JoachimHenze) [reply base/shell/explorer/traywnd.cpp:3049]: @learn-more yes, I did orienedt at all the surrounding functions here that returned LRESULT as TRUE/FALSE.
Do you want me to change them all? Or can I push it like that? I don't have any preference in that regard.

Comment (learn-more) [reply base/shell/explorer/traywnd.cpp:3049]: TBH this PR already cost more time than it will ever gain.

**Pull request merged**